### PR TITLE
Adding more questions to the manual's FAQ chapter

### PR DESCRIPTION
**Pull request by williln**
Feedback welcome and desired!

### CODE REVIEW

File: faq/README.md [hunk 1]
```diff
@@ -47,3 +47,23 @@ that are "mixed gender," but are really mostly men. We've found that by limiting
 our workshop to a majority of women participants, women are more comfortable
 signing up, asking questions, getting involved, and even staying involved later
 on. And that's a good thing!
+
+**Q: I'm an organizer, and someone is saying that my Django Girls event breaks discrimination laws. What do I do?** 
+
+A: First of all, we are so sorry this is happening to you. 
+
+Most of the time when people make this accusation, it's because they don't understand the law. Django Girls events, even when limited to only women, do not break the law, any more than women's only hacker schools like [Hackbright](http://hackbrightacademy.com/) or [App Camp for Girls](http://appcamp4girls.com/) do. Furthermore, our events are free, and might be best likened to a Pyladies educational meetup.
```
Comment (kennethlove) [inline faq/README.md]: PyLadies

File: faq/README.md [hunk 2]
```diff
@@ -47,3 +47,23 @@ that are "mixed gender," but are really mostly men. We've found that by limiting
...
 signing up, asking questions, getting involved, and even staying involved later
 on. And that's a good thing!
+
+**Q: I'm an organizer, and someone is saying that my Django Girls event breaks discrimination laws. What do I do?** 
+
+A: First of all, we are so sorry this is happening to you. 
+
+Most of the time when people make this accusation, it's because they don't understand the law. Django Girls events, even when limited to only women, do not break the law, any more than women's only hacker schools like [Hackbright](http://hackbrightacademy.com/) or [App Camp for Girls](http://appcamp4girls.com/) do. Furthermore, our events are free, and might be best likened to a Pyladies educational meetup. 
+
+We recommend you either ignore people who are accusing you of being discriminatory, or of breaking the law; or send them one of the links in the response to the above question, "Isn't Django Girls sexist?" If you feel harassed, or are not sure how to respond, reach out to us as [hello@djangogirls.org](hello@djangogirls.org) and we would be glad to support you however we can.
```
Comment (kennethlove) [inline faq/README.md]: Probably want to make this a `mailto:` link?

`[hello@djangogirls.org](mailto:hello@djangogirls.org)`

Comment (kerstin) [inline faq/README.md]: I think I'd also link the PyLadies website (unless it was already linked to in a previous section).
(Back)Links help everyone. (:

**Comment by williln** [comment]
Thanks @kennethlove and @kerstin!

**Comment by olasitarska** [comment]
@williln I just wanted to merge that, but remembered that when we were creating a non-profit this week I found this: https://www.gov.uk/government/publications/equality-act-guidance-for-charities/equality-act-guidance-for-charities#about-the-charities-exception

Which is basically saying, that yes, non-profit can legally discriminate against sex, age, race, etc. Should we include that information/resource in the faq?

**Comment by williln** [comment]
@olasitarska Added! Thanks for that link. Since it doesn't apply worldwide, I just quoted it as "an example," but let me know if you'd like to handle it differently.

**Comment by olasitarska** [comment]
Perfect. Thanks @williln! :snake:

**Pull request merged**